modules/osdk-scorecard-store-output.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-scorecard.adoc
+
+:osdk_ver: v1.16.0
+
+:_content-type: MODULE
+[id="osdk-scorecard-store-output_{context}"]
+= Local storage of scorecard results
+
+You can enable local storage of scorecard test results by using the `scorecard` command or by modifying your `config.yaml` file.
+
+[IMPORTANT]
+====
+There is a known issue with the Operator SDK scorecard storage utility. Currently, running the `operator-sdk scorecard` command with the `--test-output` flag returns an empty directory. There is no known workaround at this time. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2017669[*BZ#2017669*])
+====
+
+Running the `scorecard` command with the `--test-output` flag stores the scorecard test results in a specified directory. If you do not specify a directory, the command stores the test results in the `test-output` directory of your working directory.
+
+.`scorecard --test-output` command syntax
+[source,terminal,subs="attributes+"]
+----
+$ operator-sdk scorecard <bundle_directory_or_image> --test-output=/<output_directory> <.>
+----
+<.> Define the output directory.
+
+To enable local storage in your configuration files, you add the following to your `config.yaml` file:
+
+.Example scorecard configuration file
+[source,terminal,subs="attributes+"]
+----
+...
+tests:
+  - entrypoint:
+    - scorecard-test
+    - <test_entrypoint> <.>
+    image: quay.io/operator-framework/scorecard-test:v1.10.0
+    labels:
+      suite: <test_suite> <.>
+      test: <test_name> <.>
+    storage:
+      spec:
+        mountPath:
+          path: /<output_directory> <.>
+----
+<.> Replace with the command and arguments that you want to call, such as `scorecared-test`.
```

Comment:
```suggestion
<.> Replace `<test_entrypoint>` with the command and arguments to run, such as `scorecared-test`.
```
Will the user have enough context to define all of these parameters? If not, you might want to circle back and add more context.